README.md

```diff
@@ -27,12 +27,57 @@ The standardized MAFs generated by `standardize_mutation_data.py` will be writte
 
 **Sequence Source** ( `-s | --sequence-source` ) : The sequence source is a required field for a standard MAF as defined by the TCGA project. Accepted values are either WGS and WXS. For GENIE, WXS is used since GENIE samples are sequenced with targeted gene panels.
 
-**Annotation Suite Scripts Directory** ( ` -p | --annotation-scripts-home` ) : needs to point to the directory where the packaged Genome Nexus Annotation Pipeline JAR and annotation suite python scripts are located:
+**Annotation Suite Scripts Directory** ( ` -p | --annotation-scripts-home` ) : needs to point to the directory where the packaged Genome Nexus Annotation Pipeline JAR and annotation suite python scripts and data files are located:
 
-- merge_mafs.py
-- standardize_mutation_data.py
-- annotator.jar
+- merge_mafs.py (located in this repository)
+- standardize_mutation_data.py (located in this repository)
+- annotator.jar - The genome nexus annotation pipeline (See how to build the jar file [here](https://github.com/genome-nexus/genome-nexus-annotation-pipeline))
+- application.properties (optional) - custom properties for the annotation pipeline. The common use case is to specify custom genome nexus url with `genomenexus.base` property.
+- AwsSsl.truststore - custom trust store. Used to hold an ssl certificate (aka HTTPS certificate) of trusted websites. In our case the trust store holds certificates of custom genome nexus instance. See how to create trust store and upload the certificate [here](#create-a-trust-store).
 
+### Create a trust store
+
+Create an empty trust store with the following command:
+
+```
+keytool -noprompt \
+    -genkey -alias secure-key \
+    -dname "CN=annotation-tools" \
+    -storepass annotation-tools \
+    -keypass annotation-tools \
+    -keyalg RSA -keystore AwsSsl.truststore
+```
+
+You can export `.cer` file for your custom genome nexus using browser.
+There are multiple articles in the internet for different browsers and operating systems.
+
+After you have the `.cer` file, you can import it to the trust store with the command like the following (replace `CERTFILENAME` with the file name of your certificate):
+```
+keytool -import -noprompt \
+    -trustcacerts \
+    -alias CERTFILENAME \
+    -file CERTFILENAME.cer \
+    -keystore AwsSsl.truststore \
+    -storepass annotation-tools
+```
```

Comment:
In addition to describing how to create the trust store, wouldn't we also need to tell users how to link the trust store into the running of the annotations? I'm imagining this is related to the line in the annotation-tools scripts where it uses java to run annotator.jar ... something like
java -Djavax.net.ssl.trustStore=...

Is this already in the annotation-tools scripts? In order to use the trust store, would users need to add it?

Reply:
Your link to the code is good. I see that the trust store argument is stored in environment variable JAVA_SSL_ARGS a few lines later and then gets referenced during the execution of the java code. So this comment of mine was misconceived because it is already handled.